helmfile.d/snippets/defaults.yaml

```diff
@@ -51,6 +51,9 @@ environments:
             enabled: true
             adminUsername: admin
             theme: otomi
+            idp:
```

Comment:
could you clarify why these defaults are needed?

Reply:
they're just defaults and that section always exists as keycloak is mandatory